### ISSUE TITLE
Expired token, immediately after successful login

### ISSUE THREAD

**Issue by docbill** [forcedotcom/cli#1186]
I have seen this repeatedly now.

```
Successfully authorized briemers@redhat.com.briemers with org ID 00D8D0000008heWUAQ
14:46:11.975 sfdx force:auth:web:login --setalias briemers --instanceurl https://redhat--briemers.my.salesforce.com --setdefaultusername
 ended with exit code 0

Starting SFDX: Open Default Org

14:46:22.650 sfdx force:org:open
WARNING: apiVersion configuration overridden at "52.0"
ERROR running force:org:open:  Error authenticating with the refresh token due to: expired access/refresh token
14:46:23.969 sfdx force:org:open
 ended with exit code 1
```

I have tried sfdx auth:logout, removing the entry from ~/.sfdx, revoking all the existing credentials, etc.

**Comment by docbill** [comment]
After running an strace, I found this is the same issue as the DNS failures...  Only the environmental variable does not work around the the issue.  Nor does setting the API version.

It appears node.js is actually reading the /etc/resolv.conf, and only trying the very first entry.   Which means if I have VPN running, it only checks the DNS lookup via the VPN.   

The workaround is to disconnect VPN.  Then authorize the org.  Finally connect at least one.   

After that VPN can be connected again with no further issues that I have detected (yet).
